winui/ComboBox/Getting-Started.md

```diff
@@ -109,7 +109,7 @@ this.Content = comboBox;
 
 ## Populating items using data binding
 
-The `ComboBox` can be bound to an external data source using the [ItemsSource](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfDropDownListBase.html#Syncfusion_UI_Xaml_Editors_SfDropDownListBase_ItemsSource) property. Now, let us create Model and ViewModel classes to populate `ComboBox` with SocialMedia details.
+The `ComboBox` can be bound to an external data source using the [ItemsSource](https://help.syncfusion.com/cr/winui#Syncfusion_UI_Xaml_Editors_SfDropDownListBase_ItemsSource/Syncfusion.UI.Xaml.Charts.html) property. Now, let us create Model and ViewModel classes to populate `ComboBox` with SocialMedia details.
```

Comment:
Is the link loaded? Check valid url in below, 
https://help.syncfusion.com/cr/winui
<img width="969" alt="image" src="https://github.com/syncfusion-content/winui-docs/assets/44014283/741b0f4d-e8cc-4df4-845b-b6f6dcb3aeaa">